docs/modules/vertiq_23xx_family.rst

```diff
@@ -0,0 +1,56 @@
+
+.. _vertiq_23xx_family:
+
+*********************************
+Vertiq 23-XX Family 
+*********************************
+
+.. csv-table:: Vertiq 23-XX Family of Modules
+        :header: "Size", "Kv", "Default Firmware", "Available Firmware"
+        :align: left
+
+        "23-06", "220", "Servo", "Servo, Step/Direction"
+        "23-06", "2200", "Speed", "Speed, Pulsing"
+        "23-14", "920", "Speed", "Servo, Speed"
+        
+
+.. sidebar:: Vertiq 23-06
+
+    .. image:: ../_static/IQ2306_2200kv.png
+        :alt: Vertiq 23-06 2200Kv
```

Comment:
Purely aesthetic, but not sure how I feel about this image being on the right instead of in the center. Maybe it would be better just being under the table and intro text?